pages/testpage.tsx

```diff
@@ -1,13 +1,16 @@
-﻿import { NextPage } from 'next';
-import Layout from "../components/Layout/layout";
-import scss from "../pageStyles/index.module.scss";
+﻿import {NextPage} from 'next';
 import React from "react";
+import Layout from "../components/Layout/layout";
+import CandidateTestView from "../components/Tests/CandidateTestView";
+import {AddTwoIntegersTest} from "../components/Tests/AddTwoIntegers";
 
 const TestPage: NextPage = () => {
     return (
         <Layout>
-            <section className={scss.content}> </section>
+            <div>Test</div>
+            <CandidateTestView props={AddTwoIntegersTest}/>
```

Comment:
How do we make this page load different tests? ie - how should it know what test to display?